Sources/Kitura/Kitura.swift

```diff
@@ -22,7 +22,20 @@ import Dispatch
 
 // MARK Kitura
 
-/// A set of helper functions to make it easier to create, start, and stop Kitura based servers.
+/**
+A set of helper functions to make it easier to create, start, and stop Kitura based servers.
+### Usage Example: ###
+```swift
+let router = Router()
+let port: Int = 8080
+public func run() throws{
+    ...
+    Kitura.addHTTPServer(onPort: port, with: router)
+    Kitura.run()
+}
+```
+In this example, we create a run function inside a server Application.swift file, which will create a Kitura server on the specified port, using the given router and then run the created server.
```

Comment:
The example text needs to go above the code snippet.

Reply:
@tunniclm or @rolivieri -  I notice that we are saying two different things in this file - in the example code we call Kitura run() function inside a server Application.swift file, however, within the run() function we have a note: This function never returns - it should be the last call in your main.swift file.
Where is the recommended place to put the Kitura run() call as I'd like our usage example to reflect this (or definitely to have some better wording as to why either location may be suitable)?

Reply:
The following sentences need full stops at the end - "Stop all registered servers", "Start all registered servers and return"

Reply:
/// - Parameter unregister: If servers should be unregistered after stopped (default true).

Please add the words - they are - so it reads unregistered after they are stopped.